.teamcity/src/main/kotlin/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -42,6 +42,10 @@ val m2CleanScriptWindows = """
     )
 """.trimIndent()
 
+val cleanAndroidUserHomeScriptUnixLike = """
+    rm -rf ~/.android
```

Comment:
I think we should be more specific here and only wipe out only the `debug.keystore` and `debug.keystore.lock` file.

And why the `-f`? `-r` should be good enough, right?

Reply:
What if `~/.android` doesn't exist?

Reply:
I think we should not have the build polluting the global `~/.android`. The `~/.android` directory also includes a build cache directory, which might also break things between Gradle/AGP versions:

```
tcagent1@dev108:~/agent/work/e67123fb5b9af0ac$ ls ~/.android
analytics.settings  build-cache  build-cache.lock  cache  debug.keystore  debug.keystore.lock
```

Reply:
Hmm... I wonder if we can change the location of `~/.android` like we can with `~/.gradle`. Do you know that?